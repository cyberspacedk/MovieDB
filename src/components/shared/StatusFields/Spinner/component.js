import React from 'react';
import { Layout, Row, Col, Spin } from 'antd';

const { Content } = Layout;

const Spinner = () => (
  <Layout>
    <Content>
      <div className="top-margin">
        <Row type="flex" justify="center">
          <Col>
            <Spin />
          </Col>
        </Row>
      </div>
    </Content>
  </Layout>
);

export default Spinner;
