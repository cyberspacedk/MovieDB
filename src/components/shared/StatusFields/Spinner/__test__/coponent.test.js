import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import Spinner from '../component';

describe('Component: Spinner', () => {
  it('Match its snapshot.', () => {
    const wrapper = shallow(<Spinner />);
    expect(shallowToJson(wrapper)).toMatchSnapshot();
  });
});
