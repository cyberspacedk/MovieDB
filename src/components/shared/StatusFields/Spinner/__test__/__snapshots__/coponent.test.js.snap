// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: Spinner Match its snapshot. 1`] = `
<Adapter>
  <Adapter>
    <div
      className="top-margin"
    >
      <Row
        gutter={0}
        justify="center"
        type="flex"
      >
        <Col>
          <Spin
            size="default"
            spinning={true}
            wrapperClassName=""
          />
        </Col>
      </Row>
    </div>
  </Adapter>
</Adapter>
`;
